docs/source/plotting/MeshPlotHelp.rst

```diff
@@ -31,12 +31,17 @@ Mesh Plots
 ==========
 
 **Mesh Plots can only be accessed with a script, not through the Workbench interface**
+Here the mesh is plotted as a Poly3DCollection `Polygon <https://matplotlib.org/stable/tutorials/toolkits/mplot3d.html#polygon-plots>`_.
+
+These sample shapes can be created with :ref:`algm-SetSample` and copied using :ref:`algm-CopySample`.
```

Comment:
I think LoadSampleShape and LoadSampleEnvironment should also be mentioned here as ways of setting up a mesh sample shape